src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecution.java

```diff
@@ -109,7 +109,7 @@ public FlowDurabilityHint getDurabilityHint() {
     /**
      * Should be called by the flow owner after it is deserialized.
      */
-    public /*abstract*/ void onLoad(FlowExecutionOwner owner) throws IOException {
```

Comment:
Changes to `throws` clauses like this break source compatibility with `CpsFlowExecution` and need to be reverted:

```
-------------------------------------------------------------
COMPILATION ERROR : 
-------------------------------------------------------------
workflow-cps-plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java:[721,17] onLoad(org.jenkinsci.plugins.workflow.flow.FlowExecutionOwner) in org.jenkinsci.plugins.workflow.cps.CpsFlowExecution cannot override onLoad(org.jenkinsci.plugins.workflow.flow.FlowExecutionOwner) in org.jenkinsci.plugins.workflow.flow.FlowExecution
  overridden method does not throw java.io.IOException
1 error
-------------------------------------------------------------
```

Reply:
Indeed this is wrong and should be reverted.